src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IUsingStatement.cs

```diff
@@ -2931,5 +2932,82 @@ void M()
 
             VerifyFlowGraphAndDiagnosticsForTest<BlockSyntax>(source, expectedGraph, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact, WorkItem(32100, "https://github.com/dotnet/roslyn/issues/32100")]
+        public void UsingDeclaration_Flow()
+        {
+            string source = @"
+
+using System;
+class P : IDisposable
+{
+    void M()
+    /*<bind>*/{
+        System.Console.WriteLine(""before"");
+        using var x = new P();
```

Comment:
>using var x = new P(); [](start = 8, length = 22)

Please add tests:
- with multiple locals declared by the same statement. 
- with a control flow in the first initializer
- with a control flow in the second initializer